src/client/src/rt-components/platform/adapters/browser/browser.ts

```diff
@@ -11,4 +12,13 @@ export default class Browser implements PlatformAdapter {
 
     open: (config: WindowConfig, onClose: () => void) => openBrowserWindow(config, onClose),
   }
+
+  notification = {
+    notify: (message: any) => {
+      const { tradeNotification } = message
```

Comment:
Would be better to define a type for message that types `tradeNotifications`

Reply:
Added type for trade notification